src/main/java/mate/academy/onlinebookstore/service/shoppingcart/impl/CartItemServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.onlinebookstore.service.shoppingcart.impl;
+
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import mate.academy.onlinebookstore.dto.shoppingcart.CreateCartItemRequestDto;
+import mate.academy.onlinebookstore.dto.shoppingcart.UpdateCartItemRequestDto;
+import mate.academy.onlinebookstore.exception.EntityNotFoundException;
+import mate.academy.onlinebookstore.mapper.CartItemMapper;
+import mate.academy.onlinebookstore.model.CartItem;
+import mate.academy.onlinebookstore.model.ShoppingCart;
+import mate.academy.onlinebookstore.repository.book.BookRepository;
+import mate.academy.onlinebookstore.repository.shoppingcart.CartItemRepository;
+import mate.academy.onlinebookstore.service.shoppingcart.CartItemService;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class CartItemServiceImpl implements CartItemService {
```

Comment:
This service is redundant, all logic related to shop cart should be in ShopCartService